dataactbroker/handlers/accountHandler.py

```diff
@@ -447,7 +450,7 @@ def changeStatus(self,system_email):
     def listUsers(self):
         """ List all users ordered by status. Associated request body must have key 'filter_by' """
         user = self.interfaces.userDb.getUserByUID(LoginSession.getName(flaskSession))
-        isAgencyAdmin = True if self.interfaces.userDb.hasPermission(user, "agency_admin") else False
+        isAgencyAdmin = self.userManager.hasPermission(user, "agency_admin") and not self.userManager.hasPermission(user, "website_admin")
```

Comment:
This accounts for the situation where they are both. If they are both, the website admin permissions should have higher priority and therefore return all users and not just users within their agency.